src/CadViewerManifold.tsx

```diff
@@ -154,7 +162,7 @@ const CadViewerManifold: React.FC<CadViewerManifoldProps> = ({
         >
           <AnyCadComponent
             cad_component={cad_component}
-            circuitJson={circuitJson}
+            circuitJson={circuitJson as any}
```

Comment:
Don't add `any`, either the `circuitJson` or the `children` one of them must be a required. I don't see the handling of undefined if both of them is not present